lib/peeps.rb

```diff
@@ -0,0 +1,26 @@
+require 'pg'
+
+class Peeps
+  def self.all
+    connection = PG.connect(dbname: 'peep_manager')
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'peeps_manager_test')
+    else
+      connection = PG.connect(dbname: 'peeps_manager')
+    end
+
+    result = connection.exec("SELECT * FROM peeps;")
+    result.map { |peeping| peeping['peeps'] }
+  end
+
+  def self.create(peeps:)
+    connection = PG.connect(dbname: 'peep_manager')
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
Have you thought about extracting this from the Peeps class (checking if to use the test database) and creating your own ORM tool (Object Relation Mapping) with a persistent connection?